lib/pbench/test/unit/server/conftest.py

```diff
@@ -12,6 +12,12 @@
 [pbench-server]
 pbench-top-dir = {TMP}/srv/pbench
 
+[elasticsearch]
+server = http://elasticsearch.perf.lab.eng.bos.redhat.com:9280
```

Comment:
The test environment should not be referencing internal instances.  Let's remove these and find another way to do this.